airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -90,7 +90,7 @@
           {
             "title": "Encrypted (verify certificate)",
             "description": "Verify and use the certificate provided by the server.",
-            "required": ["ssl_method", "trustStoreName", "trustStorePassword"],
+            "required": ["ssl_method"],
```

Comment:
unless I'm missing something, the previous required fields didn't even exist in the spec.json. Removing them...